cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBridgeCluster.java

```diff
@@ -0,0 +1,519 @@
+
+        if (tls != null) {
+            List<CertSecretSource> trustedCertificates = tls.getTrustedCertificates();
+
+            if (trustedCertificates != null && trustedCertificates.size() > 0) {
+                for (CertSecretSource certSecretSource : trustedCertificates) {
+                    // skipping if a volume mount with same Secret name was already added
+                    if (!volumeMountList.stream().anyMatch(vm -> vm.getName().equals(certSecretSource.getSecretName()))) {
+                        volumeMountList.add(createVolumeMount(certSecretSource.getSecretName(),
+                                TLS_CERTS_BASE_VOLUME_MOUNT + certSecretSource.getSecretName()));
+                    }
+                }
+            }
+        }
+
+        if (tlsAuthCertAndKey != null) {
+            // skipping if a volume mount with same Secret name was already added
+            if (!volumeMountList.stream().anyMatch(vm -> vm.getName().equals(tlsAuthCertAndKey.getSecretName()))) {
+                volumeMountList.add(createVolumeMount(tlsAuthCertAndKey.getSecretName(),
+                        TLS_CERTS_BASE_VOLUME_MOUNT + tlsAuthCertAndKey.getSecretName()));
+            }
+        } else if (passwordSecret != null)  {
+            volumeMountList.add(createVolumeMount(passwordSecret.getSecretName(),
+                    PASSWORD_VOLUME_MOUNT + passwordSecret.getSecretName()));
+        }
+
+        return volumeMountList;
+    }
+
+    public Deployment generateDeployment(Map<String, String> annotations, boolean isOpenShift, ImagePullPolicy imagePullPolicy, List<LocalObjectReference> imagePullSecrets) {
+        DeploymentStrategy updateStrategy = new DeploymentStrategyBuilder()
+                .withType("RollingUpdate")
+                .withRollingUpdate(new RollingUpdateDeploymentBuilder()
+                        .withMaxSurge(new IntOrString(1))
+                        .withMaxUnavailable(new IntOrString(0))
+                        .build())
+                .build();
+
+        return createDeployment(
+                updateStrategy,
+                Collections.emptyMap(),
+                annotations,
+                getMergedAffinity(),
+                getInitContainers(imagePullPolicy),
+                getContainers(imagePullPolicy),
+                getVolumes(isOpenShift),
+                imagePullSecrets);
+    }
+
+    @Override
+    protected List<Container> getContainers(ImagePullPolicy imagePullPolicy) {
+
+        List<Container> containers = new ArrayList<>();
+
+        Container container = new ContainerBuilder()
+                .withName(name)
+                .withImage(getImage())
+                .withCommand("/opt/strimzi/bin/kafka_bridge_run.sh")
+                .withEnv(getEnvVars())
+                .withPorts(getContainerPortList())
+                .withLivenessProbe(ModelUtils.createHttpProbe(livenessPath, REST_API_PORT_NAME, livenessProbeOptions))
```

Comment:
@ppatierno We have hardcoded 8081 in the bridge, right?

Reply:
no you can configure it, through the `HEALTH_SERVER_PORT` env var which is 8081 by default.